src/keygen/participant.rs

```diff
@@ -628,6 +647,8 @@ mod tests {
 
     impl KeygenParticipant {
         pub fn new_quorum<R: RngCore + CryptoRng>(
+            sid: Identifier,
+            input: (),
```

Comment:
suggestion: Since this is always `()` you can ditch this parameter and hardcode the `()` in L669.

Reply:
Was this left undone intentionally?